src/main/java/se/sundsvall/smloader/integration/db/model/CaseEntity.java

```diff
@@ -19,9 +17,11 @@ public class CaseEntity {
 	@Id
 	private String id;
```

Comment:
Is this supposed to be an UUID or can it be converted to a long instead?

Reply:
I have not decided yet if it should be an UUID or flowInstanceId (with prefix for showing if it's internal or external case)

Reply:
Well, maybe it's just me, but I would prefer to have the primary key generation internal and not let another system be responsible for the generation of these ID:s.